api/v2.0/swagger.yaml

```diff
@@ -1189,6 +1189,31 @@ paths:
           $ref: '#/responses/404'
         '500':
           $ref: '#/responses/500'
+  /icons/{digest}:
+    get:
+      summary: Get artifact icon
+      description: Get the artifact icon with the specified digest
+      tags:
+        - icon
+      operationId: getIcon
+      parameters:
+        - $ref: '#/parameters/requestId'
+        - $ref: '#/parameters/digest'
+      responses:
+        '200':
+          description: Success
+          headers:
+            Cache-Control:
+              description: The cache control header to control the expiration time of the icon
+              type: string
+          schema:
+            $ref: '#/definitions/Icon'
+        '400':
```

Comment:
401 and 403?

Reply:
This is a public service that sharing the same icon (with the same digest) among all the projects.